src/features/transaction-wizard/mappers/as-description-list-items.tsx

```diff
@@ -270,12 +270,16 @@ const asKeyRegistrationTransaction = (transaction: BuildKeyRegistrationTransacti
       dt: 'Registration',
       dd: transaction.online ? onlineKeyRegistrationLabel : offlineKeyRegistrationLabel,
     },
-    ...(params.voteKey ? [{ dt: 'Voting key', dd: Buffer.from(params.voteKey).toString('base64') }] : []),
-    ...(params.selectionKey ? [{ dt: 'Selection key', dd: Buffer.from(params.selectionKey).toString('base64') }] : []),
-    ...(params.stateProofKey ? [{ dt: 'State proof key', dd: Buffer.from(params.stateProofKey).toString('base64') }] : []),
-    ...(params.voteFirst ? [{ dt: 'First voting round', dd: params.voteFirst }] : []),
-    ...(params.voteLast ? [{ dt: 'Last voting round', dd: params.voteLast }] : []),
-    ...(params.voteKeyDilution ? [{ dt: 'Vote key dilution', dd: params.voteKeyDilution }] : []),
+    ...('voteKey' in params && params.voteKey ? [{ dt: 'Voting key', dd: Buffer.from(params.voteKey).toString('base64') }] : []),
+    ...('selectionKey' in params && params.selectionKey
+      ? [{ dt: 'Selection key', dd: Buffer.from(params.selectionKey).toString('base64') }]
+      : []),
+    ...('stateProofKey' in params && params.stateProofKey
+      ? [{ dt: 'State proof key', dd: Buffer.from(params.stateProofKey).toString('base64') }]
+      : []),
+    ...('voteFirst' in params && params.voteFirst !== undefined ? [{ dt: 'First voting round', dd: params.voteFirst }] : []),
+    ...('voteLast' in params && params.voteLast ? [{ dt: 'Last voting round', dd: params.voteLast }] : []),
+    ...('voteKeyDilution' in params && params.voteKeyDilution ? [{ dt: 'Vote key dilution', dd: params.voteKeyDilution }] : []),
```

Comment:
I'm wondering if this bit can be 
```
    ...('voteKey' in params
      ? [
          { dt: 'Voting key', dd: Buffer.from(params.voteKey).toString('base64') },
          { dt: 'Selection key', dd: Buffer.from(params.selectionKey).toString('base64') },
          ...(params.stateProofKey ? [{ dt: 'State proof key', dd: Buffer.from(params.stateProofKey).toString('base64') }] : []),
          { dt: 'First voting round', dd: params.voteFirst },
          { dt: 'Last voting round', dd: params.voteLast },
          { dt: 'Vote key dilution', dd: params.voteKeyDilution },
        ]
      : []),
```

Reply:
We could, however I did it like this so that if a value is present it's displayed. That way it ensures what the user is previewing is what'll be sent.